### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by burnash**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+- [demo_weaviate.ipynb](https://github.com/dlt-hub/weaviate_demo/blob/main/demo_weaviate.ipynb): shows how to load data from PDF files, specifically invoices, into Weaviate.
+- [sql_to_weaviate.ipynb](https://github.com/dlt-hub/weaviate_demo/blob/main/sql_to_weaviate.ipynb): shows how to import data from a public MySQL database into Weaviate.
+
+## Prerequisites
+
+To run these notebooks, you will need credentials of a Weaviate instance.
+
+## License
+
+This repository is licensed under the [Apache License 2.0](https://github.com/dlt-hub/weaviate_demo/blob/main/LICENSE.txt). Please refer to the `LICENSE.txt` file for more details.
```
Comment (AstrakhantsevaAA) [inline README.md:17]: well now we need `LICENSE.txt` file  XD

File: README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Weaviate Data Loading Demos
+
+This repository contains Jupyter notebooks that illustrate various methods for loading data into a Weaviate database 
+using the [dlt](https://github.com/dlt-hub/dlt) library.
+
+## Repository Contents
+
+- [demo_weaviate.ipynb](https://github.com/dlt-hub/weaviate_demo/blob/main/demo_weaviate.ipynb): shows how to load data from PDF files, specifically invoices, into Weaviate.
+- [sql_to_weaviate.ipynb](https://github.com/dlt-hub/weaviate_demo/blob/main/sql_to_weaviate.ipynb): shows how to import data from a public MySQL database into Weaviate.
```
Comment (AstrakhantsevaAA) [inline README.md:9]: can you add zendesk demo colab (you can convert it to JN)

Comment (AstrakhantsevaAA) [reply README.md:9]: and we can add info how to open JN from github in colab https://colab.research.google.com/github/googlecolab/colabtools/blob/master/notebooks/colab-github-demo.ipynb

Comment (burnash) [reply README.md:9]: I'll add it in a next PR.

**Pull request merged**